ivy_tests/test_ivy/test_frontends/test_torch/test_loss_functions.py

```diff
@@ -563,6 +563,79 @@ def test_torch_nll_loss(
     )
 
 
+# gaussian_nll_loss
+@handle_frontend_test(
+    fn_tree="torch.nn.functional.gaussian_nll_loss",
+    dtype_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_value=0.01,
+        max_value=1.0,
+        allow_inf=False,
+        min_num_dims=1,
```

Comment:
The right approach would be to use `get_dtypes("valid") here, and if you get an error that a dtype is not supported for the native torch function, then you add that dtype as unsupported for our frontend function as well https://lets-unify.ai/ivy/deep_dive/data_types.html#supported-and-unsupported-data-types.

Reply:
The same goes for `target` and `var` as well.

Reply:
Fixed these.